lightmd/render/tests/html_render_test.py

```diff
@@ -130,18 +148,46 @@ def test_list_wrapper_render(content, ordered, html):
     list_wrapper = ListWrapper(content, is_ordered=ordered)
     assert html == list_wrapper.render(get_html_format)
 
-@pytest.mark.parametrize("content, html, children", [
-    (r'abc', r"<blockquote>abc</blockquote>", [TextBlock('abc')]),
-    (r'<html> rand tag </html>',
-     r"<blockquote><html> rand tag </html></blockquote>", [TextBlock('<html> rand tag </html>')]),
-])
+
+@pytest.mark.parametrize("content, html, children",
+                         [(r'abc',
+                           r"<blockquote>abc</blockquote>",
+                           [TextBlock('abc')]),
+                             (r'<html> rand tag </html>',
+                              r"<blockquote><html> rand tag </html></blockquote>",
```

Comment:
why <blockquote><html>?

Reply:
This change was caused by autopep8. I assume the original purpose is to test blockquote with multiple content.